draft-ietf-mls-extensions.md

```diff
@@ -881,16 +906,18 @@ described in {{safe-hpke}} with an empty context. Other functions are defined in
 The sender MUST set the authentication scheme to
 `TargetedMessageAuthScheme.HPKEAuthPsk`.
 
-As described in {{safe-hpke}} the `hpke_context` is a LabeledExtensionContent struct
-with the following content, where `group_context` is the serialized context of
-the group.
+The `hpke_context` is an TargetedMessageContext struct with the following
+content, where `group_context` is the serialized context of the group.
 
 ~~~ tls
-label = "MLS 1.0 ExtensionData"
-extension_type = ExtensionType
-extension_data = group_context
-~~~
+struct {
+  opaque label<V>;
+  opaque context<V>;
+} TargetedMessageContext;
 
+label = "MLS 1.0 TargetedMessageData"
+context = group_context
```

Comment:
This should likely change, but I'll do a follow-up PR. The group context is public, meaning you don't have to be a member to know it. I'll try to see if we can replace the AuthPSK mode with that, but I'm not sure yet.

Anyway, we can merge this, and I'll rewrite it later.